tests/src/ExistingSite/ContactFormTest.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_showcase\ExistingSite;
+
+/**
+ * Contact form tests.
+ */
+class ContactFormTest extends ShowcaseExistingSiteTestBase {
```

Comment:
```suggestion
class IntegrationTest extends ShowcaseExistingSiteTestBase {
```

IntegrationTest becasue we are testing the integration of the modules into showcase